src/main/java/seedu/duke/template/LcTemplate.java

```diff
@@ -85,7 +85,9 @@ protected void setCapacitor(double value) {
      */
     @Override
     public void setComponent(String s, double value) {
-        assert s.equals("c") || s.equals("l");
+
+        assert s.equals("l") || s.equals("c");
+
```

Comment:
Any particular reason for including so many blank newlines haahaha

Reply:
My bad :(